### DISCUSSION TITLE
Belajar dasar github

### DISCUSSION THREAD

**Discussion by ghost** [community/community#81917] [New to GitHub]
Selamat malam

**Comment by faridnec** [comment]
Malam, kak kalau mau belajar github aku rekomendasi apply Student Pack Developer terus dari sana mereka langsung ajarin hands-on github learning. Aku dah coba dan lumayan efektif. Belajarnya modelnya langsung main sama github seperti commit, pull request, branch, markdown dll. Kalo nggak di YouTube ada banyak sumber juga sbg contoh [see](https://youtu.be/RGOj5yH7evk?si=aEZV0mwDdrr1yg9N)

Semoga membantu!

**Comment by Akash1134** [comment]
Hi @Togieeee , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
